Update endpoints that need to be exposed (#27798)
fleetdm/fleet: articles/what-api-endpoints-to-expose-to-the-public-internet.md

## Using Fleet Desktop on remote devices

> If you are using Fleet Desktop and want the **My device** page to work, `/api/*/fleet/device/*` must be exposed.

Author: ```suggestion
If you are using Fleet Desktop `/api/*/fleet/device/*` and `/assets/*`must be exposed (for the end user's **Fleet Desktop > My device** page).
```

> If you're using Fleet Desktop `/api/*/fleet/device/*` and `/api/*/assets/*` must be exposed (for the end user **Fleet Desktop > My device** page).

Reviewer: ```suggestion
If you're using Fleet Desktop `/api/*/fleet/device/*` must be exposed in the API, and for the end user **Fleet Desktop > My device** page `/device/*` and `/assets/*` must be exposed.
```

Author: Looks good! Added those changes. Please feel free to merge if everything looks good 🚀

> If you are using Fleet Desktop `/api/*/fleet/device/*` and `/api/*/assets/*`must be exposed (for the end user's **Fleet Desktop > My device** page).

Author: ```suggestion
If you're using Fleet Desktop `/api/*/fleet/device/*` and `/api/*/assets/*`must be exposed (for the end user's **Fleet Desktop > My device** page).
```

> If you're using Fleet Desktop `/api/*/fleet/device/*` and `/api/*/assets/*` must be exposed (for the end user's **Fleet Desktop > My device** page).

Author: ```suggestion
If you're using Fleet Desktop `/api/*/fleet/device/*` and `/api/*/assets/*` must be exposed (for the end user **Fleet Desktop > My device** page).
```
